modules/platforms/dotnet/examples/Apache.Ignite.Examples/Datagrid/MultiTieredCacheExample.cs

```diff
@@ -0,0 +1,136 @@
+namespace Apache.Ignite.Examples.Datagrid
+{
+    using System;
+    using System.IO;
+    using System.Threading;
+    using Apache.Ignite.Core;
+    using Apache.Ignite.Core.Cache;
+    using Apache.Ignite.Core.Cache.Configuration;
+    using Apache.Ignite.Core.Cache.Eviction;
+    using Apache.Ignite.Core.Discovery.Tcp;
+    using Apache.Ignite.Core.Discovery.Tcp.Multicast;
+    using Apache.Ignite.Core.SwapSpace.File;
+
+    /// <summary>
+    /// This example demonstrates how multi-tiered Ignite cache stores data in various ways.
+    /// NOTE: There must be no other cluster nodes running on the host.
+    /// <para />
+    /// 1) Build the project Apache.Ignite.ExamplesDll (select it -> right-click -> Build).
+    ///    Apache.Ignite.ExamplesDll.dll must appear in %IGNITE_HOME%/platforms/dotnet/examples/Apache.Ignite.ExamplesDll/bin/${Platform]/${Configuration} folder.
+    /// 2) Set this class as startup object (Apache.Ignite.Examples project -> right-click -> Properties ->
+    ///     Application -> Startup object);
+    /// 3) Start example (F5 or Ctrl+F5).
+    /// </summary>
+    public class MultiTieredCacheExample
+    {
+        /// <summary>Example cache name.</summary>
+        private const string CacheName = "dotnet_multi_tiered_example_cache";
+
+        /// <summary>Cache entry size, in bytes..</summary>
+        private const int EntrySize = 1024;
+
+        [STAThread]
+        public static void Main()
+        {
+            Console.WriteLine();
+            Console.WriteLine(">>> Lifecycle example started.");
+
+            // Configure swap in the current bin directory (where our assembly is located).
+            var binDir = Path.GetDirectoryName(typeof(MultiTieredCacheExample).Assembly.Location);
+            var swapDir = Path.Combine(binDir, "ignite-swap");
+
+            Console.WriteLine(">>> Swap space directory: " + swapDir);
+
+            var cfg = new IgniteConfiguration
+            {
+                DiscoverySpi = new TcpDiscoverySpi
+                {
+                    IpFinder = new TcpDiscoveryMulticastIpFinder
+                    {
+                        Endpoints = new[] { "127.0.0.1:47500" }
+                    }
+                },
+                SwapSpaceSpi = new FileSwapSpaceSpi
+                {
+                    BaseDirectory = swapDir
+                }
+            };
+
+            using (var ignite = Ignition.Start(cfg))
+            {
+                var cacheCfg = new CacheConfiguration
+                {
+                    Name = CacheName,
+                    Backups = 1,
+                    EvictionPolicy = new LruEvictionPolicy
+                    {
+                        // Maximum number of entries that will be stored in Java heap. 
+                        MaxSize = 10
+                    },
+                    OffHeapMaxMemory = EntrySize * 10, // Limit off-heap to 10 entries.
+                    EnableSwap = true
```

Comment:
Let's add this comment line to the right of "EnableSwap = true" statement:
// Data will be swapped to disk if there is no more space in off-heap space.